iogt/static/css/layout/_l-header.scss

```diff
@@ -0,0 +1,21 @@
+.header {
+  background: $victoria;
+  color: #fff;
+  overflow: hidden;
```

Comment:
can you make this `overflow: auto` instead? Please let me know if I also did this in another site.